pjsip-apps/src/swig/pjsua2.i

```diff
@@ -121,6 +121,7 @@ using namespace pj;
 %feature("director") FindBuddyMatch;
 %feature("director") AudioMediaPlayer;
 %feature("director") AudioMediaPort;
+%feature("director") PendingJob;
```

Comment:
If this is only for Python, shouldn't we enable the SWIG director for Python only?
Even perhaps disable/remove the `PendingJob` class & `utilAddPendingJob()` method on other languages?